api/metadata_endpoints.py

```diff
@@ -26,7 +26,11 @@
 from grpc_getter import get_ts_ag_request
 
 import datastore_pb2 as dstore
-from utilities import get_unique_values_for_metadata, seconds_to_iso_8601_duration, convert_cm_to_m
+from utilities import (
+    get_unique_values_for_metadata,
+    seconds_to_iso_8601_duration,
+    convert_cm_to_m,
+)
```

Comment:
I would prefer to avoid this syntax. I thought we had linting that requires each to be on a separate line?

Reply:
I also thought we had that. Strange, will fix it.